docs/api/serviceorchestration/service-orchestration-push-management-generic-http_dynamic.md

```diff
@@ -0,0 +1,350 @@
+```
+
+The **error codes** are `400` if the request is malformed, `401` if the requester authentication was unsuccessful,
+`403` if the authenticated requester has no permission and
+`500` if an unexpected error happens. The error response also contains an
+[ErrorResponse](../data-models/error-response.md) JSON encoded body.
+
+```
+{
+  "errorMessage": "Subscription request list is empty.",
+  "errorCode": 400,
+  "exceptionType": "INVALID_PARAMETER",
+  "origin": "POST /serviceorchestration/orchestration/mgmt/push/subscribe"
+}
+```
+### unsubscribe
+
+The service operation **request** requires an [identity related header or certificate](../authentication_policy.md/#http) and a query parameter _ids_, which is a List<[UUID](../primitives.md#uuid)>. It contains the identifiers of the subscription records to remove.
+
+```
+DELETE /serviceorchestration/orchestration/mgmt/push/unsubscribe?ids=d2fefc6a-f563-40a2-9ce4-3512c2887755&ids=a44ab333-cfb5-420b-a7cf-b327904e243b HTTP/1.1
+Authorization: Bearer <identity-info>
+```
+The service operation **responds** with the status code `200` if called successfully. The success response does not contain any response body.
+
+The **error codes** are `400` if the request is malformed, `401` if the requester authentication was unsuccessful, `403` if the requester has no permission and `500` if an unexpected error happens. The error response also contains an [ErrorResponse](../data-models/error-response.md) JSON encoded body.
+
+```
+{
+  "errorMessage": "a44ab333-cfb5-420b-a7cf-b327904e243b is not owned by the requester",
+  "errorCode": 403,
+  "exceptionType": "FORBIDDEN",
+  "origin": "POST /serviceorchestration/orchestration/mgmt/push/unsubscribe"
+}
+```
+
+### trigger
+
+The service operation **request** requires an [identity related header or certificate](../authentication_policy.md/#http) and a [ServiceOrchestrationPushTriggerRequest](../data-models/service-orchestration-push-trigger-request.md) JSON encoded body.
+
+```
+POST /serviceorchestration/orchestration/mgmt/push/trigger HTTP/1.1
+Authorization: Bearer <identity-info>
+
+{
+   "targetSystems":[
+      "TemperatureConsumer"
+   ],
+   "subscriptionIds":[
+      
+   ]
+}
+```
+
+The service operation **responds** with the status code `201` if orchestration jobs were successfully created. The response also contains a [ServiceOrchestrationJobListResponse](../data-models/service-orchestration-job-list-response.md) JSON encoded body.
+
+```
+{
+   "jobs":[
+      {
+         "id":"1fac8a1c-aa4b-456e-b501-01289035fcc6",
+         "status":"IN_PROGRESS",
+         "type":"PUSH",
+         "requesterSystem":"TemperatureSensorManager",
+         "targetSystem":"TemperatureConsumer",
+         "serviceDefinition":"kelvinInfo",
+         "subscriptionId":"d2fefc6a-f563-40a2-9ce4-3512c2887755",
+         "message":"",
+         "createdAt":"2025-10-05T11:30:14Z",
+         "startedAt":"2025-10-05T11:30:17Z",
+         "finishedAt":"2025-10-05T11:31:11Z"
```

Comment:
Is the job already finished when this response is created?

Reply:
nope :) there is 11:30 vs. 11:31.
Anyway, I'm correcting this for less time: 2025-10-05T11:30:22Z